data/mods/My_Sweet_Cataclysm/sweet_comestibles.json

```
@@ -219,5 +219,28 @@
     "container": "null",
     "color": "red",
     "description": "A swirly confectionary treat in strings of sweet tasting candy.  You think you've seen them wriggle a bit.  How strange…"
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "pumpkin",
+    "name": "pumpkin",
+    "symbol": "%",
+    "copy-from": "pumpkin",
+    "rot_spawn": "GROUP_EGG_PUMPKIN",
+    "rot_spawn_chance": 100,
+    "container": "null",
+    "color": "yellow",
+    "description": "A large vegetable, about the size of your head.  Not very tasty raw, but is great for cooking, this one seems alive, weird..."
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "candy_cane",
+    "name": { "str_sp": "Candy Cane" },
+    "copy-from": "candy_cane",
+    "rot_spawn": "GROUP_EGG_CANE",
+    "rot_spawn_chance": 100,
+    "container": "null",
+    "color": "red",
+    "description": "A cane shaped candy traditionally found during winter season, it has a strange evil aura to it, almost as if it's not normal."
```

```suggestion
    "description": "A cane shaped candy traditionally found during winter season.  It has a strange aura to it, you'd swear you've seen wriggle a little bit."
```
Candies are not evil, they're just candy.